resources/assets/veganoption/lang/en_US.lang

```diff
@@ -29,6 +29,28 @@
 # Creative Tab
 itemGroup.VeganOption=The Vegan Option
 
+# WAILA and The One Probe strings
+VeganOption.waila.format=%1$s : %2$s
+VeganOption.top.format=%1$s: %2$s
+VeganOption.context.growth.title=Growth
+VeganOption.context.growth.value=%1$d%%
+VeganOption.context.growth.mature.title=Growth
+VeganOption.context.growth.mature.value=Mature
+VeganOption.context.retting.title=Retting
+VeganOption.context.retting.value=%1$d%%
+VeganOption.info.retting.not.submerged=Needs to be submerged in water
+VeganOption.info.retted=Retted
+VeganOption.info.composter.empty=Empty
+VeganOption.context.composting.title=Composting
+VeganOption.context.composting.value=%1$d%%
+VeganOption.context.temperature.title=Temperature
+VeganOption.context.temperature.value=%1$d°C
+VeganOption.info.basin.empty=Empty
+VeganOption.info.basin.open=Open
+VeganOption.info.basin.closed=Closed
+VeganOption.context.fluid.title=%1$s
+VeganOption.context.fluid.value=%2$dmB
```

Comment:
Because in order for the context strings to work correctly for all cases we have to pass `params` to every `translate` call (see LangHelper). As such, the name of the fluid is `1`, while the amount is `2`.